tools/pgsql_user_sync/proxysql_pgsql_user_sync.py

```
@@ -0,0 +1,934 @@
+            except Exception:
+                raise _sync_failure("unable to apply ProxySQL user changes") from None
+        if plan.requires_load:
+            try:
+                admin.load_runtime()
+            except Exception:
+                raise _sync_failure("unable to load ProxySQL users to runtime") from None
+            loaded = True
+            if config.sync.save_to_disk:
+                try:
+                    admin.save_to_disk()
+                except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)